templates/network-environment_v2.yaml.j2

```diff
@@ -24,13 +24,22 @@ parameter_defaults:
 {% if ceph_enabled %}
     CephStorageNetworkConfigTemplate: '/home/stack/virt/network/vlans/ceph-storage.j2'
 {% endif %}
+{% if manila_enabled %}
     ControllerParameters:
```

Comment:
this should be ControllerStorageNfsParameters:

Reply:
same here as well add checks for ControllerStorageNfs
https://github.com/redhat-performance/jetpack/blob/d9fc1892885d421619f9285bda99a44651d73bdc/templates/network-environment_v2.yaml.j2#L16
https://github.com/redhat-performance/jetpack/blob/d9fc1892885d421619f9285bda99a44651d73bdc/templates/network-environment_v2.yaml.j2#L4

Reply:
yes i've added that too with this change, deployment in progress.

Reply:
added the changes

Reply:
this is still missing can you please add it

```
{% if manila_enabled %}
       OS::TripleO::ControllerStorageNfs::Net::SoftwareConfig: OS::Heat::None
{% else %}
       OS::TripleO::Controller::Net::SoftwareConfig: OS::Heat::None 
{% endif %}
```

Reply:
thanks for pointing